Sources/Jobs/JobsConfiguration.swift

```diff
@@ -2,14 +2,18 @@ import Foundation
 import Vapor
 
 /// A `Service` to configure `Job`s
-public struct JobsConfig {
+public struct JobsConfiguration {
     
     /// Type storage
     internal var storage: [String: AnyJob]
     
+    /// Scheduled Job Storage
+    internal var scheduledStorage: [String: ScheduledJobStorage]
```

Comment:
Should `ScheduledJobStorage` be `AnyScheduledJob`?

Reply:
I don't think so because `ScheduledJob`s don't need to store data - they just need to have a single function that can then dispatch future jobs if they'd like